ci-operator/config/openshift/gcp-filestore-csi-driver-operator/openshift-gcp-filestore-csi-driver-operator-main.yaml

```diff
@@ -102,6 +102,10 @@ resources:
       memory: 200Mi
 test_binary_build_commands: make test
 tests:
+- as: unit
+  commands: make test-unit
+  container:
+    from: src
```

Comment:
Do you want this in the other branches too, or just `main` for now?

Reply:
Fixed!